import sys

def fatalError(reason):
    sys.exit("Fatal Error: %s" % reason)
